.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2.1
+
+executors:
+  node:
+    parameters:
+      tag:
+        type: string
+    docker:
+      - image: cimg/node:<< parameters.tag >>
+
+commands:
+  install-dependencies:
+    steps:
+      - run:
+          name: "Install project dependencies"
+          command: npm i
+  apt-update:
+    steps:
+      - run:
+          name: "APT update"
+          command: sudo apt-get update
+  install-libscret:
+    steps:
+      - run:
+          name: "Fetch and install libsecret"
+          command: sudo apt-get install -y libsecret-1-dev
+  run-unit-tests:
+    steps:
+      - run:
+          name: "Run unit tests"
+          command: npm run test:ci
+  run-linter:
+    steps:
+      - run:
+          name: "Run linter"
+          command: npm run lint
+
+jobs:
+  pr-validator:
+    resource_class: small
+    environment:
+      NODE_OPTIONS: --max-old-space-size=14336
+      CI: ""
+    parameters:
+      node-version:
+        type: string
+    executor:
+      name: node
+      tag: << parameters.node-version >>  
+    steps:
+      - checkout
+      # - apt-update
+      # - install-libscret
+      # - install-dependencies
+      - run:
+          name: "I will fail"
+          command: exit 1
+      # - run-linter      
+      # - run-unit-tests
+      # - when:
+      #     condition:
+      #       equal: [<< parameters.node-version >>, lts]
+      #     steps:
+      #       - run:
+      #           name: "CODECOV"
+      #           command: npm run coverage -- --token=$CODECOV_TOKEN
+
+    
+workflows:
+  ci:
+    jobs:
+      - pr-validator:
+          matrix:
+            parameters:
+              node-version: [lts, "10.24"]
```

Comment:
Can we just do `10,12,14,lts`? Do you have to provide the minor version too?

Reply:
As per docs [here](https://circleci.com/developer/images/image/cimg/node?utm_medium=SEM&utm_source=gnb&utm_campaign=SEM-gnb-DSA-Eng-ni&utm_content=&utm_term=dynamicSearch-&gclid=CjwKCAjwu5CDBhB9EiwA0w6sLb_9chwxtqm8Yh88cDohOh7TWY62a6AOmLJZKvNgBfPSir9sFtVq0RoCoRYQAvD_BwE): you need to provide minor as well, but I will give it a go to see if it works without it

Reply:
Sadly, need to specify minor as well